gamestonk_terminal/economy/economy_controller.py

```diff
@@ -157,83 +205,630 @@ def call_quit(self, _):
+            """,
+        )
+        parser.add_argument(
+            "-g",
+            "--group",
+            nargs="+",
+            type=str,
+            default="Sector",
+            dest="group",
+            help="Data group (sector, industry or country)",
+            choices=self.l_GROUPS,
+        )
+        try:
+            if other_args:
+                if "-" not in other_args[0]:
+                    other_args.insert(0, "-g")
+                other_args = [other_args[0], " ".join(other_args[1:])]
+
+            ns_parser = parse_known_args_and_warn(parser, other_args)
+            if not ns_parser:
+                return
+
+            finviz_view.view_group_data(
+                group=ns_parser.group,
+                data_type="spectrum",
+                export="",
+            )
+            # Due to Finviz implementation of Spectrum, we delete the generated spectrum figure
+            # after saving it and displaying it to the user
+            os.remove(ns_parser.group + ".jpg")
+
+        except Exception as e:
+            print(e, "\n")
+
+    def call_rtps(self, other_args: List[str]):
+        """Process rtps command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="rtps",
+            description="""
+                Real-time and historical sector performances calculated from
+                S&P500 incumbents. Pops plot in terminal. [Source: Alpha Vantage]
+            """,
+        )
+        parser.add_argument(
+            "--raw",
+            action="store_true",
+            dest="raw",
+            help="Only output raw data",
+        )
+        parser.add_argument(
+            "--export",
+            choices=["csv", "json", "xlsx"],
+            default="",
+            type=str,
+            dest="export",
+            help="Export dataframe data to csv,json,xlsx file",
+        )
+        try:
+            ns_parser = parse_known_args_and_warn(parser, other_args)
+            if not ns_parser:
+                return
+
+            alphavantage_view.realtime_performance_sector(
+                raw=ns_parser.raw,
+                export=ns_parser.export,
+            )
+
+        except Exception as e:
+            print(e, "\n")
+
+    def call_series(self, other_args: List[str]):
+        """Process series command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            prog="series",
+            description="""
+                Display (multiple) series from https://fred.stlouisfed.org. [Source: FRED]
+            """,
+        )
+        parser.add_argument(
+            "-i",
+            "--id",
+            dest="series_id",
+            required="-h" not in other_args,
+            type=str,
+            help="FRED Series ID from https://fred.stlouisfed.org. For multiple series use: series1,series2,series3",
+        )
```

Comment:
This technically doesn't use the series ID only.  Since the way it is allows for new keyword inputs

Reply:
Agreed!

> Industry: Didn't we agree to scrap the simply wall st command?

So, I was actually starting the refactoring work when I tried that command, and it's quite interesting. You can see that webpage without having any API. I think is quite good, like Finviz map. And since it was already implemented, thought why not? 

Do you disagree?